apps/crn-frontend/src/tags/TagsList.tsx

```diff
@@ -1,18 +1,23 @@
+import { EntityResponses } from '@asap-hub/algolia';
 import { TagsPageBody } from '@asap-hub/react-components';
 import { usePagination, usePaginationParams, useSearch } from '../hooks';
-import { useResearchOutputs } from '../shared-research/state';
+import { useTagSearch } from './state';
 
-const Tags: React.FC<Record<string, never>> = () => {
+type TagsProps = {
+  entities: Array<keyof EntityResponses['crn']>;
+};
+const Tags: React.FC<TagsProps> = ({ entities }) => {
   const { tags, searchQuery, filters } = useSearch();
   const { currentPage, pageSize } = usePaginationParams();
-  const { items, total } = useResearchOutputs({
+  const { items, total } = useTagSearch(entities, {
     searchQuery,
     filters,
     currentPage,
     pageSize,
     noResultsWithoutCriteria: true,
```

Comment:
We probablly don't need this as a prop because it will be the behaviour for all tag searches